front/app/api/internal_comments/types.ts

```diff
@@ -1,40 +1,36 @@
-import { IRelationship, Multiloc, ILinks } from 'typings';
+import { IRelationship, ILinks } from 'typings';
 import { Keys } from 'utils/cl-react-query/types';
 import commentsKeys from './keys';
 
-export type CommentsKeys = Keys<typeof commentsKeys>;
+export type InternalCommentsKeys = Keys<typeof commentsKeys>;
 
-export type ICommentParameters = {
+export type IInternalCommentParameters = {
   ideaId?: string;
   initiativeId?: string;
   userId?: string;
```

Comment:
Odd that we have `userId` and we are using `author_id` everywhere else. Not sure it is a problem but highlighting it